packages/expo-modules-core/ExpoModulesCore.podspec

```diff
@@ -1,16 +1,21 @@
 require 'json'
 
+absolute_react_native_path = 'react-native'
+if !ENV['REACT_NATIVE_PATH'].nil?
+  absolute_react_native_path = File.expand_path(ENV['REACT_NATIVE_PATH'], Pod::Config.instance.project_root)
+end
+
 unless defined?(install_modules_dependencies)
   # `install_modules_dependencies` and `add_dependency` are defined in react_native_pods.rb.
   # When running with `pod ipc spec`, these methods are not defined and we have to require manually.
-  require File.join(File.dirname(`node --print "require.resolve('react-native/package.json')"`), "scripts/react_native_pods")
+  require File.join(File.dirname(`node --print "require.resolve('#{absolute_react_native_path}/package.json')"`), "scripts/react_native_pods")
```

Comment:
if the `absolute_react_native_path` is already an absolute path, we don't necessarily to use `node --print "require.resolve()"` to resolve the path again.

Reply:
Yeah, you're right, I just changed this a bit in the case that the `REACT_NATIVE_PATH` env is not available